src/stack.js

```diff
@@ -1,5 +1,37 @@
 'use strict'
 
 export default class Stack {
-  // your code here
+  constructor(){
```

Comment:
Overall nitpick I'm seeing: spaces before braces (has a nice ring to it, no?). Be consistent with the style you adopt (I personally prefer lots of whitespace because it makes for more readable code).